File: TelerikUIExamples/TelerikUIExamples/PersonListViewCell.m
//
//  PersonListViewCell.m
//  TelerikUIExamples
//
//  Copyright © 2015 Telerik. All rights reserved.
//

#import "PersonListViewCell.h"

@implementation PersonListViewCell
{
    CAGradientLayer* _gradient;
}

- (instancetype)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        self.contentView.backgroundColor = [UIColor colorWithRed:0.91 green:0.91 blue:0.91 alpha:1.0];
        self.textLabel.numberOfLines = 0;
        self.textLabel.lineBreakMode = NSLineBreakByWordWrapping;
        self.textLabel.textAlignment = NSTextAlignmentCenter;
        self.textLabel.font =[UIFont fontWithName:@"HelveticaNeue-Italic" size:16];
        self.textLabel.layer.cornerRadius = 10;
        self.textLabel.layer.masksToBounds = YES;
    }
    return self;
}

- (void)layoutSubviews
{
    [super layoutSubviews];
    self.imageView.frame = CGRectMake(0 , 10, 120, 100);
    self.textLabel.frame = CGRectMake(0, self.imageView.frame.size.height, self.frame.size.width, 60);
}

@end

File: TelerikUIExamplesInSwift/TelerikUIExamplesInSwift/PersonListViewCell.swift
//
//  PersonListViewCell.swift
//  TelerikUIExamplesInSwift
//
//  Copyright © 2015 Telerik. All rights reserved.
//

class PersonListViewCell: TKAutoCompleteSuggestionCell {

    override init(frame: CGRect) {
        super.init(frame: frame)
        
        self.contentView.backgroundColor = UIColor(red: 0.91, green: 0.91, blue: 0.91, alpha: 1.00)
        self.textLabel.numberOfLines = 0;
        self.textLabel.lineBreakMode = NSLineBreakMode.ByWordWrapping
        self.textLabel.textAlignment = NSTextAlignment.Center
        self.textLabel.font = UIFont(name:"HelveticaNeue-Italic", size:16)
        self.textLabel.layer.cornerRadius = 10
        self.textLabel.layer.masksToBounds = true
    }
    
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        fatalError("init(coder:) has not been implemented")
    }
    
    override func layoutSubviews() {
        super.layoutSubviews()
        
        self.imageView.frame = CGRectMake(0 , 10, 120, 100);
        self.textLabel.frame = CGRectMake(0, self.imageView.frame.size.height, self.frame.size.width, 60);
    }
}
